base/Enums.jl

```diff
@@ -29,7 +29,7 @@ end
 """
     @enum EnumName EnumValue1[=x] EnumValue2[=y]
 
-Create an [`Enum`](:obj:`Enum`) type with name `EnumName` and enum member values of
+Create an `Enum` type with name `EnumName` and enum member values of
```

Comment:
`Enum` itself doesn't actually have a docstring, only `@enum` does. For some reason sphinx didn't ever complain about that.